release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Added new asset configuration to allow for GCC / GCC High environments [PAPP-27308]
```

Comment:
Add point for 'Added encryption for the sensitive values stored in the state file'

Reply:
Added notes in release note.